core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -130,6 +145,7 @@
     private VirtualHost defaultVirtualHost;
     private EventLoopGroup workerGroup = CommonPools.workerGroup();
     private boolean shutdownWorkerGroupOnStop;
+    private final Map<ChannelOption<?>, Object> serverChannelOptions = new LinkedHashMap<>();
```

Comment:
You also need to keep the `channelOptions` which can be set using `ServerBootStrap.childOption()`.

Reply:
How about just `channelOptions` for consistency with `ServerBootstrap`?